stable/kong/README.md

```diff
@@ -69,7 +69,9 @@ and their default values.
 | env                               | Additional [Kong configurations](https://getkong.org/docs/latest/configuration/)               |
 | runMigrations                     | Run Kong migrations job                                                | `true`                |
 | readinessProbe                    | Kong readiness probe                                                   |                       |
+| readinessProbe.httpGet.schema     | Schema used by readyness probe. Set to HTTP if admin.useTLS = false    | `HTTPS`               |
```

Comment:
Actual value [for these](https://github.com/kubernetes/charts/blob/master/stable/kong/values.yaml#L61) is `scheme` not _schema_ (`e` vs `a`)

Reply:
True! 😱